### PR TITLE
Update docs on cluster verification

### PR DESCRIPTION
**Pull request by cxbrowne1207**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/clustermgmt/observability/cluster-verify.md [hunk 1]
```diff
@@ -8,40 +8,117 @@ date: 2017-01-05
...
 ```
 
 You may also check the status of the cluster control plane resource directly. 
 This can be especially useful to verify clusters with multiple control plane nodes after an upgrade.
+
 ```
-kubectl get kubeadmcontrolplanes.controlplane.cluster.x-k8s.io
-NAME                       INITIALIZED   API SERVER AVAILABLE   VERSION              REPLICAS   READY   UPDATED   UNAVAILABLE
-supportbundletestcluster   true          true                   v1.20.7-eks-1-20-6   1          1       1
+kubectl get kubeadmcontrolplanes.controlplane.cluster.x-k8s.io -n eksa-system $CLUSTER_NAME
```
Comment (tatlat) [inline docs/content/en/docs/clustermgmt/observability/cluster-verify.md:94]: nit: not too important but maybe add an instruction to export CLUSTER_NAME as well

File: docs/content/en/docs/clustermgmt/observability/cluster-verify.md [hunk 2]
```diff
@@ -8,40 +8,117 @@ date: 2017-01-05
...
+To verify that the expected number of cluster nodes are up and running, use the `kubectl` command to show that nodes are `Ready`.
+
+This will confirm that the expected number of nodes are present.
+Worker nodes are named using the cluster name followed by the worker node group name (example: my-cluster-md-0), the others are control plane nodes.
+
+```
+kubectl get nodes
+NAME                              STATUS   ROLES           AGE   VERSION
+mgmt-clrt4                        Ready    control-plane   3d22h   v1.27.1-eks-61789d8
+mgmt-md-0-5557f7c7bxsjkdg-l2kpt   Ready    <none>          3d22h   v1.27.1-eks-61789d8
```
Comment (chrisnegus) [inline docs/content/en/docs/clustermgmt/observability/cluster-verify.md:67]:
```suggestion
kubectl get nodes
NAME                                    STATUS   ROLES           AGE   VERSION
my-cluster-clrt4                        Ready    control-plane   3d22h   v1.27.1-eks-61789d8
my-cluster-md-0-5557f7c7bxsjkdg-l2kpt   Ready    <none>          3d22h   v1.27.1-eks-61789d8
```

**Review by chrisnegus — COMMENTED** [review]
This is nice information to add. I just have a comment about using consistent cluster naming throughout the doc.

File: docs/content/en/docs/clustermgmt/observability/cluster-verify.md (hunk 1 above)
Comment (cxbrowne1207) [reply docs/content/en/docs/clustermgmt/observability/cluster-verify.md:94]: I was suppose to remove `$CLUSTER_NAME` here actually. The output shown gives all the kubeadmcontrolplanes 

Thanks

**Review by jiayiwang7 — APPROVED** [review]

**Pull request merged**